apps/files/src/components/FileList.vue

```diff
@@ -249,16 +261,8 @@ export default {
       return this.actionsInProgress.some(itemInProgress => itemInProgress.id === item.id)
     },
 
-    $_disabledActionTooltip(item) {
-      if (this.$_actionInProgress(item)) {
```

Comment:
Do not remove existing functionality pls. If it's broken open a ticket 😉

Reply:
alright!